charts/argo-templates/templates/poll-job.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
If we're going to have a separate top-level template for this, it should be wrapped in an `if`-`else` condition like all the others. It'll be more complex because it needs to be rendered if ANY template that calls it is rendered.